doc/api/process.md

```diff
@@ -2212,6 +2212,18 @@ process to have `root` or the `CAP_SETGID` capability.
 
 The `groups` array can contain numeric group IDs, group names or both.
```

Comment:
Not directly relevant to this PR, but optionally add a serial comma here to be consistent with our documentation style guide.

```suggestion
The `groups` array can contain numeric group IDs, group names, or both.
```